src/afl-fuzz-queue.c

```diff
@@ -921,6 +921,9 @@ u32 calculate_score(afl_state_t *afl, struct queue_entry *q) {
 
   switch (afl->schedule) {
 
+    case DUMB:
```

Comment:
Is there a difference with the `EXPLORE` schedule?

Reply:
This is kind of weird: https://github.com/AFLplusplus/AFLplusplus/blob/22e2362f0fd5685548696f487639104a0059e3eb/src/afl-fuzz-queue.c#L92. Not quite sure what's going on here...

Reply:
Maybe it would be more safe to simply make the fast schedule worse (this seems to be the default now). For instance, we could change this line:
https://github.com/AFLplusplus/AFLplusplus/blob/22e2362f0fd5685548696f487639104a0059e3eb/src/afl-fuzz-queue.c#L970

To something like `if (1) break;`.

Reply:
This is much simpler. Thanks!